jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby9k-json-vs-pson/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+node {
+    checkout scm
+    pipeline = load 'jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy'
+}
+
+pipeline.multipass_pipeline([
+        [
+                job_name: 'oss-latest-jruby-9k-json',
+                gatling_simulation_config: '../simulation-runner/config/scenarios/foss5x-medium-1250-2-hours.json',
+                server_version: [
+                        type: "oss",
+                        version: "latest"
+                ],
+                agent_version: [
+                        version: "latest"
+                ],
+                code_deploy: [
+                        type: "r10k",
+                        control_repo: "git@github.com:puppetlabs/puppetlabs-puppetserver_perf_control.git",
+                        basedir: "/etc/puppetlabs/code/environments",
+                        environments: ["production"],
+                        hiera_config_source_file: "/etc/puppetlabs/code/environments/production/root_files/hiera.yaml"
+                ],
+                background_scripts: [
+                        "./jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh"
+                ],
+                archive_sut_files: [
+                        "/var/log/puppetlabs/puppetserver/metrics.json"
+                ],
+                jruby_jar: "/opt/puppetlabs/server/apps/puppetserver/jruby-9k.jar",
+        ],
+        [
+                job_name: 'oss-latest-jruby9k-pson',
+                gatling_simulation_config: '../simulation-runner/config/scenarios/foss25x-medium-1250-2-hours.json',
```

Comment:
Versioning these solely per the Puppet APIs, i.e., /puppet/v3/catalog, may not work here.  For example, for the PSON->JSON migration, /puppet/v3 is used for both but only the use of HTTP headers, etc., changed.

Might be a bit more verbose, but we could name these files with both versions of the master and agent - e.g., foss-agent-4x-master-5x-...  We could also qualify the names if necessary, like foss-agent-4x-master-5x-pson...  I still like being able to name the files such that the intention can be derived without a separate keyring, if possible.  Open to any specific suggestions you have on naming, though, since we definitely haven't had a very consistent / understandable strategy around these in the past.